config/canonical.json

```
@@ -2452,6 +2452,8 @@
     "tags": {
       "amenity": "bank",
       "brand": "ING",
+      "brand:wikidata": "Q645708",
+      "brand:wikipedia": "en:ING_Group",
```

Can you change this to remove the underscore?  Thanks!
```suggestion
      "brand:wikipedia": "en:ING Group",
```

---

Yes - technically what OpenStreetMap likes in this value is called a [local interlanguage link](https://en.wikipedia.org/wiki/Help:Interlanguage_links#Local_links).  The format is just the wiki language code, a colon, and the page title (underscores in the url become spaces in the title).

---

Thank you, nice to know that! Updated right now.